package.json

```diff
@@ -57,6 +67,7 @@
         "graphql-tools": "~1.1.0",
         "graphql-type-json": "~0.1.4",
         "inflection": "~1.12.0",
+        "lodash": "^4.17.4",
```

Comment:
Not used anywhere. Besides, if some lodash functions are needed, just import them as independent packages as we did for merge.

Reply:
Removed